test/command_line/helper/command_runner.rb

```diff
@@ -182,7 +182,34 @@ def normalize_init_line(line)
     end
   end
 
+  def normalize_groonga_log(text)
+    normalized = ""
+    text.split("\n").each do |line|
+      line = normalize_init_line(line)
+      if line =~ /grn_init: <VERSION>/ or
+        line =~ /grn_fin / or
+        line =~ /vm.overcommit_memory / or
+        line =~ /spec:\d+/ or
+        line =~ /DDL:\d+/ or
+        line =~ /To set vm.overcommit_memory/ or
+        line =~ /run 'sudo \/sbin\/sysctl/
```

Comment:
We should not remove. We should normalize. We can ignore log from `groonga` not `grndb` by removing log file: https://github.com/groonga/groonga/pull/987#discussion_r293231987

Reply:
@kou 

Do you mean that 
just remove `line =~ ..` and
 add the following line(normalized output) assert_equal (as expected) is better, don't you?

```
1970-01-01 00:00:00.000000|n| grn_init: <VERSION>
...
```

Reply:
Yes.